src/HeaderSort/HeaderSort.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.listviewcontrols.HeaderSort" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Header sort</name>
+    <description>Sort Mendix list views using headers</description>
+    <icon></icon>
+    <properties>
+        <property key="entity" type="entity">
+            <caption>List view with entity</caption>
+            <category>General</category>
+            <description/>
```

Comment:
Add description `Select the same entity as the list view that this widget should interact with. If multiple list views with this entity are found, the closest is used` This description should be updated to all list view widgets. Pagination, checkbox, drop down sort, drop down filter, search etc.

Reply:
Done